src/libcore/iter/range.rs

```diff
@@ -267,14 +267,11 @@ impl<A: Step> ops::RangeFrom<A> {
     /// # Examples
     ///
     /// ```
-    /// # #![feature(step_by)]
+    /// #![feature(step_by)]
     ///
-    /// for i in (0u8..).step_by(2).take(10) {
-    ///     println!("{}", i);
-    /// }
+    /// let result: Vec<_> = (0..).step_by(2).take(5).collect();
+    /// assert_eq!(result, vec![0, 2, 4, 6, 8]);
```

Comment:
I think removing the annotation changes it from `u8` to implicitly `i32`, not that it matters.